app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorFragment.kt

```diff
@@ -75,30 +75,38 @@ class CreditCardEditorFragment : Fragment(R.layout.fragment_credit_card_editor)
             true
         }
         R.id.save_credit_card_button -> {
-            saveCreditCard()
+            view?.hideKeyboard()
+
+            val creditCard = args.creditCard
+
+            if (creditCard != null) {
```

Comment:
Instead of duplicating creating the same `UpdatableCreditCardFields` in if/else we can create it once and then use it in different methods.

Reply:
Nice!